metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -51,31 +51,56 @@ case class SbtBuildTool(version: String) extends BuildTool {
           sbtArgs
         ).flatten
     }
-    writeSbtMetalsPlugin(config)
+    removeOldPlugin()
```

Comment:
why are we removing the old plugin? Shouldn't we just stop using it and not touch the home directory unless absolutely necessary?
If there is a reason for doing this, please add it as a comment

Reply:
I think the reason is this https://github.com/scalameta/metals/pull/852#discussion_r310684012

Reply:
I will add a comment.

Reply:
Added